util/tune/current_settings.rb

```diff
@@ -40,10 +43,10 @@
 #   value = puppetserver_jruby_max_active_instances
 #
 def puppetserver_jruby_max_active_instances
-  if File.exist? PE_PUPPET_SERVER_CONF
-    line = File.open(PE_PUPPET_SERVER_CONF).grep(/max-active-instances/)[0]
-    value = line.scan(/\d+/).first unless line.nil?
-  end
+  return "NA" unless File.exist? PE_PUPPET_SERVER_CONF
```

Comment:
NA is defined as a constant at the beginning of this script.  The constant should be used here.